src/System.ServiceModel.NetTcp/ref/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "System.ObjectModel": "4.0.0",
-    "System.Runtime": "4.0.0",
+    "System.Runtime": "4.0.20",
```

Comment:
Question: does bumping this version of a product package mean we need to bump the minor version of this package?  Can you explain why this upgrade was needed?  All the other project.json touches are to tests -- this is the only one touching a product package.